client/src/components/Posts/ShowPostByID.tsx

```
@@ -0,0 +1,152 @@
+                              {/* Reverse the comments to display the latest on top */}
+                              {post.comments && post.comments.slice().reverse().map((comment: Comment) => (
+                                  <div key={comment._id} className="flex items-start gap-2 mb-4">
+                                      <div className="h-8 w-8">
+                                          <img src="/placeholder.svg?height=32&width=32" alt={comment.user_username} />
+                                      </div>
+                                      <div>
+                                          <p className="text-sm font-medium">{comment.user_username}</p>
+                                          <p className="text-sm">{comment.text}</p>
+                                      </div>
+                                  </div>
+                              ))}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)